lib/lightspeed/request.rb

```diff
@@ -108,7 +108,7 @@ def extract_rate_limits(response)
 
     def uri
       uri = self.class.base_path + @path
-      uri += "?" + URI.encode_www_form(@params) if @params && @method == :get
+      uri += "?" + URI.encode_www_form(@params) if @params && @method == :get # rubocop:disable Style/StringConcatenation
```

Comment:
i think this is string plus? so maybe doesn't need disabling?

Reply:
I had memory that `URI.encode_www_form(@params)` did something strange. I'll double check

Reply:
I think you may be remembering e.g. URI::HTTP#+

Reply:
I'm wrong - I have changed the implementation - Thanks